docs/concepts/abstractions/pod.md

```diff
@@ -0,0 +1,74 @@
+---
+---
+
+{% capture overview %}
+This page provides an overview of `Pod`, the smallest deployable object in the Kubernetes object model.
+{% endcapture %}
+
+* TOC
+{:toc}
+
+{% capture body %}
+### Understanding Pods
+
+A *Pod* is the basic building block of Kubernetes--the smallest and simplest unit in the Kubernetes object model that you create or deploy. A Pod represents a running process on your cluster.
+
```

Comment:
Multiple applications is the "multiple containers tightly coupled." I thought that was a big part of the Pod use case.